### DISCUSSION TITLE
Dialogic fails to load on when game is on itch.io

### DISCUSSION THREAD

**Discussion by senma22** [dialogic-godot/dialogic#2147] [Q&A] [answered]
Hi. I doing a school project (game development). I used Godot 3.3 and with dialogic 1.2.5. everything works perfectly in godot but when I uploaded my game on itch.io. The dialogues stopped working and the game was stuck. Please help. Thanks so much. 
![Screen Shot 2024-03-20 at 7 31 29 PM](https://github.com/dialogic-godot/dialogic/assets/118864607/87ef7782-3186-49ad-8bfe-6e41a4b77ed0)

**Comment by zaknafean** [comment, marked as answer]
Hey Senma, 

So you are definitely using some old versions of Dialogic and Godot, so I can't say for 100% certainty that it will all work, but the most common problem with exports and Dialogic 1.x is forgetting to export the json and cfg files.

![image](https://github.com/dialogic-godot/dialogic/assets/7741797/70618dc0-c167-4355-9f52-eca02d26a913)
 
 When you export, go to the Resources tab, and ensure that *.json, *.cfg is there like in the above picture. Then report back!

**Reply by senma22** [reply]
Thank you so much !!!! It worked. I appreciate it!
